app/models/job_application.rb

```diff
@@ -29,6 +30,13 @@ class JobApplication < ApplicationRecord
   enum :status, { initial: "initial", completed: "completed", uncompleted: "uncompleted", submitted: "submitted", rejected: "rejected" },
        default: :initial, validate: true
 
+  def attachment(question)
+    return photo if question.photo?
+    return cover_letter if question.cover_letter?
+
+    resume
```

Comment:
How come you've set it up like this?

Reply:
An application form has several attachments now ... and the number will probably grow.
I need to be able to retrieve that specific attachment.
I need the question as a parameter to be able to do so.